src/core/services/theming/theming.js

```diff
@@ -282,6 +357,8 @@ function ThemingProvider($mdColorPalette) {
       alwaysWatchTheme = alwaysWatch;
     },
 
+    enableBrowserColor: enableBrowserColor,
+
```

Comment:
Correct me if I'm wrong, but the `setBrowserColor()` function takes an options object, not a single color, right? So you would still have to pass an options param of theme, palette and hue? That's kinda opposite of what I was suggesting.